content/pages/meetup/2024/0911-nycu.rst

```diff
@@ -0,0 +1,125 @@
+========================================
+Meetup 2024 September 11th in NYCU
+========================================
+
+:date: 2024-09-07 13:42
+:url: meetup/2024/0911-nycu
+:save_as: meetup/2024/0911-nycu.html
+
+Date
+-----
+
+* Date: September 11th, 2024
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+There will be many different subjects discussed at the same time.
+
++-----------------+--------------------------+---------------------------------+
+| Timetable       | Topic 1                  | Topic 2                         |
```

Comment:
@Gene0315 @YenPeiChen07 I'd like to ask for your help to host the third topic for the curve processing.  Could you please add it in the agenda?  It can be done in a follow-up PR.  (@ThreeMonth03 probably wants to merge this soon.)

Reply:
> @Gene0315 @YenPeiChen07 I'd like to ask for your help to host the third topic for the curve processing. Could you please add it in the agenda? It can be done in a follow-up PR. (@ThreeMonth03 probably wants to merge this soon.)

Thanks, I will merge this pr today. Maybe we could create a new issue about the agenda of next meetup.

Reply:
> @Gene0315 @YenPeiChen07 I'd like to ask for your help to host the third topic for the curve processing. Could you please add it in the agenda? It can be done in a follow-up PR. (@ThreeMonth03 probably wants to merge this soon.)

Sure. I will add a column for it in the table today.